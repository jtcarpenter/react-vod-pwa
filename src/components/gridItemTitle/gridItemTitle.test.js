import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {GridItemTitle} from
    'components/gridItemTitle/GridItemTitle';

const mockGridItemTitle = (
    <GridItemTitle/>
)

describe('GridItemTitle', () => {
    it('should render', () => {
        const wrapper = shallow(mockGridItemTitle);
        expect(wrapper.exists()).toBe(true);
    });

    it('should display correctly', () => {
        const gridItemTitle = renderer.create(mockGridItemTitle);
        expect(gridItemTitle.toJSON()).toMatchSnapshot();
    });
});
