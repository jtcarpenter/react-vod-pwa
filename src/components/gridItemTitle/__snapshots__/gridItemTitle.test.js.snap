// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridItemTitle should display correctly 1`] = `
<h2
  className="sc-bdVaJa dRdLhg"
/>
`;
